examples/local_sgd/resnet/train_gpu.sh

```diff
@@ -29,7 +30,7 @@ IMAGE_SHAPE=3,224,224
 #gpu params
 FUSE=True
 NCCL_COMM_NUM=2
-NUM_CARDS=4
+NUM_CARDS=8
 FP16=False #whether to use float16 
```

Comment:
is this configuration applicable? in my mind, local sgd have not been tested in FP16 mode

Reply:
If it is not applicable, u'd better remove it.

Reply:
Done